cosmovisor/README.md

```diff
@@ -88,6 +88,20 @@ In order to support downloadable binaries, a tarball for each upgrade binary wil
 
 The `DAEMON` specific code and operations (e.g. tendermint config, the application db, syncing blocks, etc.) all work as expected. The application binaries' directives such as command-line flags and environment variables also work as expected.
 
+### Commands
+
+As Cosmovisor is meant to be used as a wrapper for a `Cosmos SDK` application, it does not support many commands. 
+
+However, to determine the version of Cosmovisor, the following command can be run.
```

Comment:
```suggestion
Because Cosmovisor is meant to be used as a wrapper for a Cosmos SDK application, it does not require many commands. 

To determine the version of Cosmovisor, run the following command:
```